src/seedu/addressbook/data/tagging/Tagging.java

```diff
@@ -0,0 +1,35 @@
+package seedu.addressbook.data.tagging;
+
+import java.util.List;
+
+import seedu.addressbook.data.person.Person;
+import seedu.addressbook.data.tag.Tag;
+
+public class Tagging {
+	private final Person person;
+	private final Tag tag;
+	private final Boolean isAdding;
```

Comment:
This is a reasonable choice to keep track of the addition/deletion. You can consider the alternative of defining a enumeration for OperationType (e.g.) with "ADD" and "DELETE" as values.